pom.xml

```diff
@@ -47,7 +47,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <spring.boot.version>1.5.4.RELEASE</spring.boot.version>
+        <spring.boot.version>1.5.6.RELEASE</spring.boot.version>
```

Comment:
@rajadilipkolli We're a bit concerned that our test coverage is not enough and keeping updating version might have potential bugs. How about keeping the version as it was and rename this PR as a code cleanup change?